pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 const (
+	buildInitBinary = "/layers/org.cloudfoundry.go-mod/app-binary/build-init"
```

Comment:
Maybe we should add a comment here saying that when the go buildpack changes this should be changed as well